pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -23,6 +21,7 @@ import (
 	operatorv1 "github.com/openshift/api/operator/v1"
 	operatorv1client "github.com/openshift/client-go/operator/clientset/versioned/typed/operator/v1"
 	operatorv1informers "github.com/openshift/client-go/operator/informers/externalversions/operator/v1"
+	"github.com/openshift/cluster-kube-apiserver-operator/pkg/operator/operatorclient"
```

Comment:
nit: can you please fix these imports to comply with our format?

Reply:
Done